packages/react/src/Prose/Prose.features.stories.tsx

```diff
@@ -114,3 +117,291 @@ UnorderedList.decorators = [
+<table>
+  <caption>${t('developer_growth_caption')}</caption>
+  <thead>
+    <tr>
+      <th scope="col">${t('country')}</th>
+      <th scope="col">${t('number_of_developers')}</th>
+      <th scope="col">${t('yoy_growth')}</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>${t('singapore')}</td>
+      <td>>1M ${t('developers')}</td>
+      <td>39%</td>
+    </tr>
+    <tr>
+      <td>${t('india')}</td>
+      <td>>13.2M ${t('developers')}</td>
+      <td>36%</td>
+    </tr>
+    <tr>
+      <td>${t('hong_kong_sar')}</td>
+      <td>>1.6M ${t('developers')}</td>
+      <td>35%</td>
+    </tr>
+    <tr>
+      <td>${t('vietnam')}</td>
+      <td>>1.5M ${t('developers')}</td>
+      <td>34%</td>
+    </tr>
+    <tr>
+      <td>${t('indonesia')}</td>
+      <td>>2.9M ${t('developers')}</td>
+      <td>31%</td>
+    </tr>
+    <tr>
+      <td>${t('japan')}</td>
+      <td>>2.8M ${t('developers')}</td>
+      <td>31%</td>
+    </tr>
+    <tr>
+      <td>${t('philippines')}</td>
+      <td>>1.3M ${t('developers')}</td>
+      <td>31%</td>
+    </tr>
+    <tr>
+      <td>${t('thailand')}</td>
+      <td>>857K ${t('developers')}</td>
+      <td>25%</td>
+    </tr>
+    <tr>
+      <td>${t('south_korea')}</td>
+      <td>>1.9M ${t('developers')}</td>
+      <td>22%</td>
+    </tr>
+    <tr>
+      <td>${t('australia')}</td>
+      <td>>1.4M ${t('developers')}</td>
+      <td>21%</td>
+    </tr>
+  </tbody>
+</table>
+<p>Nunc velit odio, posuere eu felis eget, consectetur fermentum nisi. Aenean tempor odio id ornare ultrices. Quisque blandit condimentum tellus, semper efficitur sapien dapibus nec.</p>
+`
+
+export const DefaultTable = args => {
+  const {t} = useTranslation('Prose')
+  return <Prose html={proseMarkup(t)} {...args} />
+}
+DefaultTable.storyName = 'With a table (default)'
+
+export const EditorialTable = () => {
+  const {t} = useTranslation('Prose')
+  return <Prose html={proseMarkup(t)} variant="editorial" />
+}
+EditorialTable.storyName = 'With a table (editorial variant)'
+
+export const DefaultTableNarrowView = () => {
+  const {t} = useTranslation('Prose')
+  return <Prose html={proseMarkup(t)} enableFullWidth />
+}
+DefaultTableNarrowView.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+DefaultTableNarrowView.storyName = 'With a table (narrow)'
+
+const dataLabelsMarkup = (t: (key: string) => string) => `
+<h2>${t('vertically_stacked_layout')}</h2>
+<p>${t('vertically_stacked_description')}</p>
+<p><strong>${t('vertically_stacked_note')}</strong></p>
+<table>
+  <caption>${t('developer_growth_caption')}</caption>
+  <thead>
+    <tr>
+      <th scope="col"><span class="visually-hidden">${t('country')}</span></th>
+      <th scope="col">${t('number_of_developers')}</th>
+      <th scope="col">${t('yoy_growth')}</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>${t('singapore')}</td>
+      <td data-label="${t('number_of_developers')}">>${t('developers')}</td>
```

Comment:
We're missing some numbers in these examples, e.g.

```suggestion
      <td data-label="${t('number_of_developers')}">>1M ${t('developers')}</td>
```

Reply:
Thanks Josh, that's fixed 👌